2020/July/README.md

```diff
@@ -0,0 +1,13 @@
+# Meetup title
```

Comment:
Can you please remove this file, that way it will be easy for me to rebase my changes.